### PR TITLE
Unbreak master and more

### PR DESCRIPTION
**Pull request by dug-stripe**
Fixes two fatal bugs in https://github.com/stripe/timberlake/pull/67 relating to destructuring global vars: `const {_} = window._;` should have been `const {_} = window;` and likewise for s/_/d3/.

Also includes a bunch of other fixes that were in flight, including fixing `no-param-reassign`, moving the JobRow components to their own file, and fixing a bunch of `var` usage.

r? @mdoan-stripe

### CODE REVIEW

File: js/jobrow.jsx [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  }
+
+  onClick(e) {
+    if (isLeftClickEvent(e) && !isModifiedEvent(e)) {
+      hashHistory.push(`/job/${this.props.job.id}`);
+    }
+  }
+
+  render() {
+    const columns = this.columns();
```
Comment (mdoan-stripe) [inline js/jobrow.jsx]: where is `columns` defined..?

Comment (mdoan-stripe) [reply js/jobrow.jsx]: Oh I see, it's implemented in a subclass

Comment (dug-stripe) [reply js/jobrow.jsx]: 🙈

**Comment by mdoan-stripe** [comment]
LGTM

File: js/jobrow.jsx (hunk 1 above)
Comment (dug-stripe) [reply js/jobrow.jsx]: i could add a default implementation that throws an exception…

**Pull request merged**